pkg/user/user_integration_test.go

```diff
@@ -167,14 +243,196 @@ func TestUserHandler(t *testing.T) {
 				var me model.User
 				client.GetJSON(t, "/me", &me, inttest.WithAuthToken(user1Token.AccessToken))
 
-				assert.Equal(t, "user1@dhis2.org", me.Email)
+				assert.Equal(t, "user6@dhis2.org", me.Email)
 			}
 
 			{
 				t.Log("GetAllIsUnauthorized")
 
 				client.Do(t, http.MethodGet, "/users", nil, http.StatusUnauthorized, inttest.WithAuthToken(user1Token.AccessToken))
 			}
+
+			{
+				t.Log("SignInCookies")
+
+				createUser(t, client, userService, "user7@dhis2.org", "oneoneoneoneoneoneone111")
+				requestBody := strings.NewReader(`{}`)
+				request := client.NewRequest(t, http.MethodPost, "/tokens", requestBody, inttest.WithBasicAuth("user7@dhis2.org", "oneoneoneoneoneoneone111"), inttest.WithHeader("Content-Type", "application/json"))
+
+				response, err := client.Client.Do(request)
+				require.NoError(t, err)
+
+				assert.Equal(t, http.StatusCreated, response.StatusCode)
+				actualCookies := response.Cookies()
```

Comment:
Can we use something like `assert.EqualValues(t, expectedCookies, response.Cookies())`? So instead of asserting on individual fields assert on a struct. This makes the test easier to read. It also prevents the whac-a-mole of I fixed one failing assertion, rerun only to find another field was also not equal, ...

Reply:
It's difficult because `Value` and `Raw` contains the token and I don't know what it is

Reply:
So you are saying you want to assert only on some but not other fields? testify does not seem to be able to do that 😢 

https://github.com/google/go-cmp can which is my preferred tool when asserting.

Then at least create a helper like

`assertCookie(t *testing.T, ....)`

This makes the test more readable as we have tons of lines just for asserting cookies 😅